pulp_docker/app/views.py

```diff
@@ -1,6 +1,49 @@
+import os
+
+from gettext import gettext as _
+from logging import getLogger, DEBUG
+
+from django.shortcuts import get_object_or_404
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist, MultipleObjectsReturned
+from django.http import (
+    HttpResponse,
+    HttpResponseRedirect,
+    HttpResponseForbidden,
+    HttpResponseNotFound,
+    StreamingHttpResponse)
+
+from wsgiref.util import FileWrapper
+
+from pulpcore.plugin.models import ContentArtifact
+from pulpcore.app.views.content import ArtifactNotFound, PathNotResolved
+
+from pulp_docker.app.models import DockerDistribution, ManifestListManifest, ManifestList, \
+    ManifestBlob, ImageManifest, Tag
+
+from rest_framework.negotiation import BaseContentNegotiation
 from rest_framework import response, views
 
 
+
+log = getLogger(__name__)
+log.level = DEBUG
```

Comment:
I haven't seen it done this way before. Is this equivalent to `log.setLevel(DEBUG)`?

Reply:
I removed this all together. We are not logging anything in here anyway.